packages/back-end/types/experiment-snapshot.d.ts

```diff
@@ -138,6 +138,7 @@ export interface SnapshotBanditSettings {
     date: Date;
     weights?: number[];
   }[];
+  alpha: number; 
```

Comment:
wondering if we should track this in ExperimentSnapshotAnalysisSettings? (although it only has consequence for bandits, I believe...)

Reply:
remove

Reply:
removed